internal/pkg/template/templates/workloads/partials/cf/env-controller.yml

```diff
@@ -7,6 +7,11 @@ EnvControllerAction:
     Workload: !Ref WorkloadName
 {{- if .Aliases}}
     Aliases: {{ fmtSlice (quoteSlice .Aliases) }}
+{{- end}}
+{{- if $.CDNEnabled}}
+    # This will not be processed by env controller, but trigger env controller to
```

Comment:
Can this cause "No updates to be performed" error from env stack?

Reply:
Should we just not use the env controller's outputs here? like don't force refresh but instead if `$.CDNEnabled` then in `LoadBalancerDNSAlias` resource we do:
```
Fn::ImportValue:
          !Sub "${App}-${Env}-CloudFrontDomainName"
```

Reply:
It will not if there's no env param diff.

Reply:
Oh that's a good idea. Let me try that.

Reply:
Ok i've tried to switch between having CDN and no CDN and I think it works!

Reply:
We'll just error out.

Reply:
There are currently two types of aliases being managed in service stack:
1. Default alias
2. Aliases that are validated by imported certificates AND have `"alias.hosted_zone"` field (i.e. user wants us to upsert the A records for them)

We've decided to error out for case 1, I would +1 to erroring out for case 2 (exactly what https://github.com/aws/copilot-cli/pull/3878#discussion_r943942248 is saying) as well 🤔  I think it's not a bad idea to stop building on top of the service stack aliases if we are not sure about the implementation. I also believe both use cases 1 and 2 are on the rarer side.